datastore/uncached.go

```diff
@@ -401,3 +401,26 @@ func (s *DataStore) GetLonghornSnapshotUncached(name string) (*longhorn.Snapshot
 func (s *DataStore) GetAllLonghornBackingImages() (runtime.Object, error) {
 	return s.lhClient.LonghornV1beta2().BackingImages(s.namespace).List(context.TODO(), metav1.ListOptions{})
 }
+
+// GetNonDefaultBackupTargets returns an uncached list of BackupTargets in
+// Longhorn namespace directly from the API server.
+// Using cached informers should be preferred but current lister doesn't have a
+// field selector.
+// Direct retrieval from the API server should only be used for one-shot tasks.
+func (s *DataStore) GetNonDefaultBackupTargets() (runtime.Object, error) {
```

Comment:
Only get non-default target? Why not get all backup targets?

Reply:
Default backup target settings are stored in the Setting CR,
but we can save all backup targets.
Modified. Thanks